### PR TITLE
Cleanup CSS in Play UI

### PR DESCRIPTION
**Pull request by alexey-milovidov**
### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Cleanup CSS in Play UI. The pixels are more evenly placed. Better usability for long content in table cells.

### CODE REVIEW

File: programs/server/play.html [hunk 1]
```diff
@@ -193,10 +205,9 @@
 
         #data-table
         {
+            width: 100%;
             border-collapse: collapse;
             border-spacing: 0;
-            /* I need pixel-perfect alignment but not sure the following is correct, please help */
```
Comment (alexey-milovidov) [inline programs/server/play.html:198]: This issue is resolved.

File: programs/server/play.html [hunk 2]
```diff
@@ -71,12 +72,24 @@
             --logo-color-active: #333;
         }
 
+        *
+        {
```
Comment (peledni) [inline programs/server/play.html:76]: nit: you don't really need this on every element, set it where needed

File: programs/server/play.html [hunk 3]
```diff
@@ -71,12 +72,24 @@
...
+            margin: 0;
+        }
+
+        html
+        {
+            /* The fonts that have full support for hinting. */
             font-family: Liberation Sans, DejaVu Sans, sans-serif, Noto Color Emoji, Apple Color Emoji, Segoe UI Emoji;
             background: var(--background-color);
             color: var(--text-color);
+            padding: 0.5rem;
```
Comment (peledni) [inline programs/server/play.html:92]: all these properties should probably be on body, not html

File: programs/server/play.html [hunk 4]
```diff
@@ -119,17 +127,21 @@
...
+        #query_div
         {
             /* Make enough space for even huge queries. */
             height: 20%;
+        }
+
+        #query
+        {
+            height: 100%;
             width: 100%;
```
Comment (peledni) [inline programs/server/play.html:139]: Maybe add: `resize: vertical;` here - I don't think you need should be able to resize horizontally

**Review by peledni — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
@mergifyio update

**Comment by alexey-milovidov** [comment]
@mergifyio update after #36591.

**Comment by alexey-milovidov** [comment]
@mergifyio update

File: programs/server/play.html [hunk 5]
```diff
@@ -253,11 +265,11 @@
             display: none;
         }
 
-        /* When mouse pointer is over table cell, will display full text (with wrap) instead of cut.
-           TODO Find a way to make it work on touch devices. */
```
Comment (alexey-milovidov) [inline programs/server/play.html:257]: Also checked - it works perfectly on iPad.

**Comment by alexey-milovidov** [comment]
@mergifyio update

**Comment by alexey-milovidov** [comment]
@mergifyio update

**Comment by alexey-milovidov** [comment]
@tavplubix is notified about the failing integration test.

**Pull request merged**